docs/docs/api/LayoutAnimations/LayoutTransitions.md

```diff
@@ -0,0 +1,107 @@
+---
+id: layoutTransitions
+title: Layout Transitions
+sidebar_label: Layout Transitions
+---
+
+## How to use layout transition?
+
+### 1. Import Layout 
+```js
+    import { Layout } from 'react-native-reanimated';
```

Comment:
Same problem here as explained earlier. `Layout` does not exists in the reanimated package.

Reply:
It does actually :D